packages/contract/lib/utils/index.js

```diff
@@ -80,6 +80,60 @@ const Utils = {
       .filter(log => log != null);
   },
 
+  consoleLog(receiptLogs) {
```

Comment:
seems like this method might could use a better name... `Utils.handleConsoleLogs` perhaps? `logConsoleLogs`?

Reply:
I'm not opposed to simply re-naming this, @CruzMolina thoughts?

Reply:
I like `logConsoleLogs`.

Reply:
What about `printConsoleLogs` lol?

Reply:
Sure, just merge the damn thing already :trollface: